tests/behat/features/mfa-totp.feature

```diff
@@ -3,10 +3,10 @@ Feature: Use MFA TOTP
   I want to use MFA TOTP
 
    Background:
-    Given the "group" "EDITOR group" has permissions "CMS_ACCESS_LeftAndMain"
+    Given the "group" "EDITOR" has permissions "CMS_ACCESS_LeftAndMain"
```

Comment:
Surely full `CMS_ACCESS_LeftAndMain` access isn't necessary for every scenario here?

Reply:
DONE